stdlib/ocaml/generate.mc

```diff
@@ -51,7 +52,44 @@ with ("abcABC____'", gensym ()).0
 utest (escapeName ("ABC123", gensym ())).0
 with ("_BC123", gensym ()).0
 
+let _seqOps = [
+  "make",
+  "empty",
+  "length",
+  "concat",
+  "get",
+  "set",
+  "cons",
+  "snoc",
+  "reverse",
+  "split_at"
+]
+
+let _seqOpHashMap =
+  let mkOp = lam op.
+    nameSym (join ["Boot.Intrinsics.Mseq.", op])
+  in
+  foldl (lam a. lam op. hashmapInsert hashmapStrTraits op (mkOp op) a)
+        hashmapEmpty _seqOps
+
+let _seqOp = lam op.
+  nvar_ (hashmapLookupOrElse
+    hashmapStrTraits
+    (lam _. error (strJoin " " ["Operation", op, "not found"]))
+    op _seqOpHashMap)
```

Comment:
The absence of a linebreak here feels weird to me

Reply:
Agree, I think it looks better now.